assets/scss/projects.scss

```diff
@@ -100,7 +100,7 @@ html.modal-open, html.modal-open body {
   }
 
   .modal {
-    z-index: 100;
+    z-index: 1000;
```

Comment:
```suggestion
    z-index: 99999999999999999999999999999999;
```

Reply:
I think the alerts need to be above it ;D